content/product-engineering/engineering/cloud/growth-and-integrations/index.md

```diff
@@ -0,0 +1,199 @@
+          </a>
+        </div>
+        <div style="text-align: center;">Erzhan Torokulov</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <a href="../../../../team/index.md#tharuntej-kandala" target="_blank" rel="noopener">
+            <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/tj.png" alt="Picture of TJ Kandala" style="background: transparent; width:128px;"/>
+          </a>
+        </div>
+        <div style="text-align: center;">TJ Kandala</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/join-us-vermillion.png" alt="Picture with CTA: Join Us" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">P.S.</div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer</div>
+        <div style="text-align: center; font-size: 12px;">(starting 01/10)</div>
+      </div>
+    </div>
+
+<div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/rob-rhyne.png" alt="Picture of Rob Rhyne" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Rob Rhyne</div>
+        <div style="text-align: center; font-size: 12px;">Temporary product manager and product designer, growth</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/join-us-vermillion.png" alt="Picture with CTA: Join Us" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Stephen Gutekanst </div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer (growth)</div>
+      </div>
+    </div>
+  </div>
+
+</section>
+
+## Growth vision
+_Sourcegraph is optimized to help users discover how powerful the tool is, as quickly as possible._
+
+This vision is necessarily broad because the growth-oriented goal for the team is the generation of Product Qualified Leads. These PQLs will generally be generated from users who find value in the product and integrate Sourcegraph into their daily routines. As such, this team will also impact the following objectives:
+
+- Improve the retention of all users
+- Increase the frequency of users from monthly active to daily active
+- Expand usage in existing accounts
```

Comment:
@bcreager @christinaforney @nicksnyder Can you please have a look at this section explicitly? It lays out a necessarily broad vision for the growth team that attempts to credit and capture the advances we'll be making as we seek to generate PQLs.

Note that this vision statement doesn't equate to additional goal setting this quarter but sets us up for a longer-term vision and goal setting that will help us report on the individual _stages_ of the PQL generation.

Reply:
I suggested an update to the vision statement to make it more concrete. The subsequent paragraphs feel like implementation details of the vision, so maybe we should label that differently.

I don't feel strongly about this, just suggestions.

Reply:
Thanks Nick. I like where we ended up with your help.